src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -300,10 +314,10 @@ Copyright (c) .NET Foundation. All rights reserved.
     <KnownFrameworkReference Include="Microsoft.AspNetCore.App"
                               TargetFramework="netcoreapp3.0"
                               RuntimeFrameworkName="Microsoft.AspNetCore.App"
-                              DefaultRuntimeFrameworkVersion="$(MicrosoftAspNetCoreAppRuntimeWinX64PackageVersion)"
-                              LatestRuntimeFrameworkVersion="$(MicrosoftAspNetCoreAppRuntimeWinX64PackageVersion)"
+                              DefaultRuntimeFrameworkVersion="$(_MicrosoftAspNetCoreApp30PackageVersion)"
+                              LatestRuntimeFrameworkVersion="$(_MicrosoftAspNetCoreApp30PackageVersion)"
                               TargetingPackName="Microsoft.AspNetCore.App.Ref"
-                              TargetingPackVersion="$(AspNetTargetingPackVersion)"
+                              TargetingPackVersion="$(_AspNet30TargetingPackVersion)"
                               RuntimePackNamePatterns="Microsoft.AspNetCore.App.Runtime.**RID**"
                               RuntimePackRuntimeIdentifiers="@(AspNetCoreRuntimePackRids, '%3B')"
```

Comment:
I'd suggest having a different RID list for each version (although as we discussed, ideally it would come as data via the insertion).